### DISCUSSION TITLE
self hosted Cloud : Failed to upload "data.gpkg": [QT-2] Connexion fermée

### DISCUSSION THREAD

**Discussion by sigsdis30** [opengisch/QField#4496] [Q&A]
Hi,
We have implemented your QFieldCloud on our server.
Our cloud runs on DEBIAN12 OS with a i5 processor, 16GB RAM and 500Go of disk space.
The configuration has been mounted under DOCKER as specified in the GitHub page and we run the docker-compose.override.test.yml.

I can't upload a qgs project or any gpkg because the Connexion is closed.
The errors are "Failed to upload "data.gpkg": [QT-2] Connexion fermée" or "Aborted "communes.gpkg" upload". We have solved the subscription problem but I don't know what causes these problems.

Can you help me ?

Thank you

**Comment by suricactus** [comment]
See https://github.com/opengisch/qfieldcloud/issues/755

**Reply by sigsdis30** [reply]
Hi, 
I put the same one on both forums. 
So that if people on this one know something, they can help :)

**Comment by sigsdis30** [comment]
Hi, 
I really need help... I can't work if that problem is not resolved...
Thanks

**Comment by suricactus** [comment]
Feel free to ask for dedicated support at sales@opengis.ch .

**Comment by nesar77** [comment]
Hi @sigsdis30 , did you solve the problem? 
I've deployed the qfieldcloud on my server; should I recompile the qfieldcloud sync plugin to change the server ip ?

**Reply by sigsdis30** [reply]
Hi @nesar77, 
Do you have an email address with which we can talk about it or maybe help each other ?
